_posts/2021-12-31-trino-2021-a-year-of-growth.md

```diff
@@ -0,0 +1,348 @@
+authorization and integration with other security systems:
+
+* Automatic configuration of TLS for 
+[secure internal communication]({{site.url}}/docs/current/security/internal-communication.html).
+This removes the need to provision per-worker TLS certificates.
+* OAuth2/OIDC opaque access tokens.
+* Configuring multiple password authentication plugins.
+* Hiding inaccessible columns from `SELECT *` statement.
+* Configuring HTTP proxy for OAuth2 authentication.
+* Handling of Server Name Indication (SNI) for multiple TLS certificates.
+
+## Data Sources
+
+### BigQuery connector
+
+* Added support for case insensitive name matching for BigQuery views.
+* Support reading `bignumeric` type whose precision is less than or equal to 
+38.
+* Add support for `CREATE SCHEMA` and `DROP SCHEMA` statements.
+* Improved support for BigQuery datetime and timestamp types.
+* Fixed failures for queries accessing `information_schema.columns` when 
+`case-insensitive-name-matching` is disabled.
+
+### Cassandra connector
+
+* Mapped Cassandra `uuid` type to Trino `uuid`.
+* Added support for Cassandra `tuple` type.
+* Changed minimum number of speculative executions from 2 to 1. 
+
+### Clickhouse connector
+
+* Added [ClickHouse connector]({{site.url}}/docs/current/connector/clickhouse.html).
+* Improved performance of aggregation queries by computing aggregations within 
+ClickHouse. Currently, the following aggregate functions are eligible for
+pushdown: `count`, `min`, `max`, `sum` and `avg`.
+* Added support for dropping columns.
+* Map ClickHouse `UUID` columns as `UUID` type in Trino instead of `VARCHAR`.
+
+### HDFS, S3, Azure and cloud object storage systems
+
+A core use case of Trino uses the Hive and Iceberg connectors to connect to
+a data lake. These connectors differ from most as Trino is the sole query engine
+as opposed to the client calling another system. Here are some changes that
+for these connectors:
+
+* Fixed queries on empty Iceberg tables without a snapshot ID that were created by Spark.
+* Updated to Iceberg 0.11.0 behavior for transforms of dates and timestamps before 1970.
+* Added procedure `system.flush_metadata_cache()` to flush metadata caches.
+* Avoid generating splits for empty files.
+* Sped up Iceberg query performance when dynamic filtering can be leveraged.
+* Increased Iceberg performance when reading timestamps from Parquet files.
+* Improved Iceberg performance for queries on nested data through dereference pushdown.
+* Added support for `INSERT OVERWRITE` operations on S3-backed tables.
+* Made the Iceberg `uuid` type available.
+* Trino views made available in Iceberg.
+
+
+### Elasticsearch connector
+
+* Added support for reading fields as `json` values.
+* Fixed failure when documents contain fields of unsupported types.
+* Added support for `scaled_float` type.
+* Added support for assuming an IAM role.
+* Fixed potential incorrect results when queries contain an `IS NULL` predicate.
```

Comment:
No need to list this one. Let's focus on features rather that bug fixes. (and maybe add a note somewhere about "fixed many, many, many bugs").